services/wordpress/wordpress.tex

```diff
@@ -0,0 +1,26 @@
+\section{Wordpress}\index{wordpress}
+
+Wordpress is a PHP content management system. It has reasonable security in a new default install, but has a poor track record for remote execution exploits. The best way to secure Wordpress is to update it if possible and remove all unnecessary or old plugins. The following will help limit damage from possible remaining exploits:
```

Comment:
The download is https://wordpress.org/latest.tar.gz and I'll add a few stuff (basically SQL commands) for the setup.